tests/foreman/cli/test_discoveryrule.py

```diff
@@ -318,178 +323,50 @@ def test_positive_delete(self, discoveryrule_factory):
         :CaseImportance: Critical
         """
         rule = discoveryrule_factory()
-        DiscoveryRule.delete({'id': rule.id})
+        target_sat.cli.DiscoveryRule.delete({'id': rule.id})
```

Comment:
We're already deleting rule in `test_positive_create_with_name` as part of teardown or can have a additional crud/e2e test for it, so can we remove this test?

Reply:
Removing